session7/src/com/itschool/homework/HomeRunChallenge7.java

```diff
@@ -3,27 +3,27 @@
 import java.util.Scanner;
 
 public class HomeRunChallenge7 {
+
     public static void main(String[] args) {
         HomeRunChallenge7 home = new HomeRunChallenge7();
         home.challengeRun7();
     }
 
     public void challengeRun7() {
+        final int LUCKY_NUMBER = 20;
```

Comment:
if you want this to be a final local variable, then the java convention is to write it with "camelCase" style. Upper SNAKE_CASE style is the convention for constants. Therefore if you want this to be a constant, then declare it at class level (before `main` method declaration) and provide all the necessary keywords:
`private static final int LUCKY_NUMBER = 20;`